server/plugin.go

```diff
@@ -19,8 +20,9 @@ type Plugin struct {
 	configuration *configuration
 }
 
+// This example demonstrates a plugin that handles HTTP requests which respond by greeting the world.
```

Comment:
Can we also become godoc compliant by writing this as follows?
```suggestion
ServeHTTP demonstrates a plugin that handles HTTP requests by greeting the world.
```

Reply:
I'm a bit surprised `golint ./...` didn't caught this one.